server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/RecipientRewriteTableMock.java

```diff
@@ -18,12 +18,18 @@
  ****************************************************************/
 package org.apache.james.transport.mailets;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang.StringUtils;
 import org.apache.james.rrt.api.RecipientRewriteTableException;
 import org.apache.james.rrt.lib.Mappings;
 import org.apache.james.rrt.lib.MappingsImpl;
 import org.apache.james.rrt.lib.MappingsImpl.Builder;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
RecipientRewriteTableMock contains this : 

```
public class RecipientRewriteTableMock implements org.apache.james.rrt.api.RecipientRewriteTable {
```
Is there anything we can do about it ?

Also, beware there is some useless javadoc at least line 28 -> 30